client/src/main/java/io/pravega/client/stream/notifications/events/CustomEvent.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream.notifications.events;
+
+import io.pravega.client.stream.notifications.Event;
+import lombok.Builder;
+
+/**
+ * Represents a customEvent.
```

Comment:
When does one of these occur?

Reply:
CustomEvent is redundant, will remove it.